lib/src/iterable_extensions.dart

```diff
@@ -643,6 +711,74 @@ extension IterableIntegerExtension on Iterable<int> {
 ///
 /// Specialized version of some extensions of [IterableNumberExtension].
 extension IterableDoubleExtension on Iterable<double> {
+  /// A minimal element of the iterable, or `null` it the iterable is empty.
+  double? get minOrNull {
+    var iterator = this.iterator;
+    if (iterator.moveNext()) {
+      var value = iterator.current;
+      while (iterator.moveNext()) {
+        var newValue = iterator.current;
+        if (newValue < value) {
+          value = newValue;
+        }
+      }
+      return value;
+    }
+    return null;
+  }
+
+  /// A minimal element of the iterable.
+  ///
+  /// The iterable must not be empty.
+  double get min {
```

Comment:
Again `=> minOrNull ?? throw StateError("No element");` and ditto for max.

Reply:
See above

Reply:
Yep, needs parentheses.